CONTRIBUTING.md

```diff
@@ -136,7 +136,8 @@ If you can, try to reproduce the highlighting problem in the text editor that th
 
 You can also try to fix the bug yourself and submit a pull-request.
 [TextMate's documentation](https://manual.macromates.com/en/language_grammars) offers a good introduction on how to work with TextMate-compatible grammars.
-You can test grammars using [Lightshow](https://github-lightshow.herokuapp.com).
+Note that Linguist uses [PCRE](https://www.pcre.org/) regular expressions, while TextMate uses [Oniguruma](https://github.com/kkos/oniguruma) library. Although they are mostly compatible there might be some differences in syntax and semantics between the two. 
```

Comment:
1. Remove the word `library` after Oniguruma, it's redundant.
~~~diff
-while TextMate uses [Oniguruma](https://github.com/kkos/oniguruma) library
+while TextMate uses [Oniguruma](https://github.com/kkos/oniguruma).
~~~

2. Place the second sentence (*"Although they are…"*) on a new line.